app/javascript/multi-stamp-tubes/components/MultiStampTubes.spec.js

```diff
@@ -268,8 +268,8 @@ describe('MultiStampTubes', () => {
     expect(wrapper.vm.targetWells).toEqual({
       A1: { pool_index: 1 },
```

Comment:
pool_index is odd (existing) naming, we're not pooling here, it's just an index for the unique barcode